applications/GeoMechanicsApplication/tests/test_benchmark_set_1.py

```diff
@@ -42,82 +42,6 @@ def test_benchmark1_1(self):
-        #
-        test_name = 'smoothrigidfootingonelasticsoil_2'
-        file_path = test_helper.get_file_path(os.path.join('.', test_name + '.gid'))
-
-        simulation = test_helper.run_kratos(file_path)
-
-        cauchy_stress_tensor = test_helper.get_cauchy_stress_tensor(simulation)
-
-
-        step_size = 0.005
-        B = 1
-        delta = 0.88
-        G = 500
-        nu=0.333
-        settlement = 0.01
-        F = settlement*2*(1+nu)*G/delta
-
-        sigma_v = rigid_footing(0, B, delta, G, nu, settlement)
-        print("sigma_v = " + str(sigma_v))
-
-        #
-        x = [i*step_size for i in range(int(B/step_size)+1)]
-        xmid = [(x[i]+x[i+1])/2 for i in range(len(x)-1)]
-
-       # sigma_v = [2 / math.pi * F / 2 / (B * math.sqrt(1 - (xi / B) ** 2)) for xi in xmid]
-
-        model_part = simulation._list_of_output_processes[0].model_part
-        elements = model_part.Elements
-
-        distances = []
-        min_distance_elements=[]
-        min_distance_idxs = []
-        for xi in xmid:
-            min_distance = 10**10
-            min_distance_element=None
-            for element_idx,element in enumerate(elements):
-
-                nodes = element.GetNodes()
-                #integration_points = element.GetIntegrationPoints()
-
-
-                centroid=[0,0,0]
-                centre_x = sum([node.X0 for node in nodes])/len(nodes)
-                centre_y = sum([node.X0 for node in nodes]) / len(nodes)
-                centre_z = sum([node.X0 for node in nodes]) / len(nodes)
-
-                centroid = [[centre_x[i],centre_y[i],centre_z[i]] for i in range(len(centre_x))]
-                for node in nodes:
-                   centroid[0] =centroid[0]+node.X0
-                   centroid[1] = centroid[1] + node.Y0
-                   centroid[2] = centroid[2] + node.Z0
-                centroid = [x/len(nodes) for x in centroid]
-               # for integration_point in integration_points:
-                #    distance = compute_distance(integration_point,[xi,0.0,0.0])
-                distance = test_helper.compute_distance(centroid, [xi, 0.0, 0.0])
-                if distance < min_distance:
-                    min_distance = distance
-                    min_distance_element = element
-                    min_distance_idx = element_idx
-            min_distance_elements.append(min_distance_element)
-            min_distance_idxs.append(min_distance_idx)
-            distances.append(min_distance)
-
-        cauchy_stresses_at_load=[cauchy_stress_tensor[min_distance_idx]for min_distance_idx in min_distance_idxs]
-        reaction_force = [(cauchy_stress_at_load[0][3]+cauchy_stress_at_load[1][3]+cauchy_stress_at_load[2][3])/3*step_size for cauchy_stress_at_load in cauchy_stresses_at_load]
-        cauchy_stress_tensor[min_distance_idxs]
-
     @KratosUnittest.skip("Test skipped, because it is unreliable. To be enabled after investigation of the root cause.")
     def test_benchmark1_4(self):
```

Comment:
I know that the names are not descriptive anyway, but do we want to keep the benchmark 1_4 and 1_5 as is, now that both 1_2 and 1_3 are gone? It's also okay with me to do that when we revisit these tests (e.g. call 1_1 test_benchmark1_biaxial_shear_stress)

Reply:
We have not found a report that uses the numbering of these benchmarks or a reason for clustering them in the 2 sets. When coming back here, I would propose to rename them completely without any numbering.